### PR TITLE
Updated ABL with New Versions of Baked Books

### PR DESCRIPTION
**Pull request by theresamasciale**
Please refer to #1775 for context. Rebaked all books and updated versions within the list (except the precal trio and college algebra with coreq). I've also added HS Physics to the list as it was missing before.

### CODE REVIEW

**Comment by lwtchu** [comment]
looks good to me?? although i'm not sure what the failed check is.

if it's fine with @rnathuji this should be good to go :eyes:

File: approved-books.json [hunk 1]
```diff
@@ -374,5 +702,13 @@
     "version": "1.9.1",
     "server": "cnx.org",
     "uuid": "e42bd376-624b-4c0f-972f-e0c57998e765"
+  }
```
Comment (rnathuji) [inline approved-books.json:705]: @theresamasciale - Ah, I think the CI is failing to parse the file as JSON because this line needs a comma at the end.

Comment (theresamasciale) [reply approved-books.json:705]: oo ok, is there a way to edit my changes so that I dont have to start all over again?

Comment (theresamasciale) [reply approved-books.json]: jk i figured it out. how about now?

**Review by rnathuji — APPROVED** [review]

**Pull request merged**